Model/Adminhtml/Messages/SocMessage.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Alma\MonthlyPayments\Model\Adminhtml\Messages;
+
+use Alma\MonthlyPayments\Helpers\ShareOfCheckout\ShareOfCheckoutHelper;
+use Magento\Framework\Notification\MessageInterface;
+use Magento\Framework\View\LayoutFactory;
+
+class SocMessage implements MessageInterface
+{
+    /**
+     * @var LayoutFactory
+     */
+    private $layoutFactory;
+    /**
+     * @var ShareOfCheckoutHelper
+     */
+    private $shareOfCheckoutHelper;
+
+    public function __construct(
+        LayoutFactory $layoutFactory,
+        ShareOfCheckoutHelper $shareOfCheckoutHelper
+    ) {
+        $this->layoutFactory = $layoutFactory;
+        $this->shareOfCheckoutHelper = $shareOfCheckoutHelper;
+    }
+
+    /**
+     * @return string
+     */
+    public function getIdentity(): string
+    {
+        return 'soc_notification';
+    }
+
+    /**
+     * @return bool
+     */
+    public function isDisplayed(): bool
+    {
+        if ($this->shareOfCheckoutHelper->getShareOfCheckoutSelectorValue() === 2) {
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        
        return $this->shareOfCheckoutHelper->getShareOfCheckoutSelectorValue() === 2;
```

`2` should probably set in an eloquent `constant`